src/coffee/mapping.coffee

```diff
@@ -367,6 +367,8 @@ class Mapping
     if _.has langH2i, attribName
       _.each langH2i[attribName], (index, language) ->
         val = row[index]
+        if not _.isString(val) || val == ""
```

Comment:
Can you explain why we doing this?

What are the use case. if we have a predefined format for searchKeywords, if that format is not followed shouldn't we return an error.

Let's discuss about this.